velox/runner/tests/CMakeLists.txt

```
@@ -23,3 +23,20 @@ target_link_libraries(
   velox_parse_parser
   velox_parse_expression
   GTest::gtest)
+
+velox_add_library(velox_query_replay_runner QueryReplayRunner.cpp)
+
+velox_link_libraries(
```

```suggestion
target_link_libraries(
```
For tests and other standalone binaries like fuzzers etc. please use the default function, so they don't get made part of libvelox.a.